run-build-functions.sh

```diff
@@ -102,6 +105,38 @@ run_yarn() {
   fi
 
 
+  if [ "$NETLIFY_YARN_WORKSPACES" = "true" ]
+  then
+    echo "NETLIFY_YARN_WORKSPACES feature flag set"
+    local workspace_output
+    local workspace_exit_code
+    # YARN_IGNORE_PATH will ignore the presence of a local yarn executable (i.e. yarn 2) and default
+    # to using the global one (which, for now, is alwasy yarn 1.x). See https://yarnpkg.com/configuration/yarnrc#ignorePath
+    workspace_output="$(YARN_IGNORE_PATH=1 yarn workspaces --json info )"
+    workspace_exit_code=$?
```

Comment:
```suggestion
    local workspace_exit_code=$?
```

(And removing the declaration above)
(Same for `workspace_output`)

Reply:
@ehmicky happy to do so for the `workspace_exit_code` however, for the previous one, declaring a `local` variable will actually override the exit code from the command we need to use afterwards - https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
TIL.
Good catch :+1:

Reply:
I owe it all to shellcheck - https://github.com/koalaman/shellcheck/ - I would be doomed writing bash/sh without it 😂